src/main/java/step3/model/Cars.java

```diff
@@ -0,0 +1,41 @@
+package step3.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static step3.utils.RacingUtils.generateRandomNumber;
+
+public class Cars {
+    private static final int BOUND = 10;
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void checkForwardConditionAndGo() {
+        for (Car car : cars) {
+            checkRandomNumber(car);
+        }
+    }
+
+    public void checkRandomNumber(Car car) {
+        int randomNumber = generateRandomNumber(BOUND);
+        car.availableForward(randomNumber);
+    }
+
+    public List<Car> getCars() {
+        return this.cars;
```

Comment:
Cars 밖에서 this.cars 에 영향을 못미치게 해주세요 :) 
```suggestion
        return Collections.unmodifiableList(this.cars);
```

or

```suggestion
        return List.copyOf(this.cars);
```

https://velog.io/@ohzzi/collection%EC%9D%98-%EB%B3%B5%EC%82%AC-new-copyOf-unmodifiable

Reply:
아..외부에서 영향을 받을 거라는 생각을 아예 하지 못했습니다..unmodifiableList로 반환하도록 수정하였습니다.